aten/src/ATen/native/quantized/TensorCompare.cpp

```diff
@@ -20,12 +20,14 @@ Tensor min_quantized_cpu(const Tensor& self) {
 
 // TODO: move to TensorMath.cpp
 
-std::tuple<Tensor, Tensor> sort_quantized_cpu(
+std::tuple<Tensor, Tensor> sort_quantized_cpu_stable(
     const Tensor& self,
+    c10::optional<bool> stable,
     int64_t dim,
     bool descending) {
   Tensor sort_int;
   Tensor sort_indicies;
+  // TODO: enable stable once signatures are added
```

Comment:
Would you elaborate on what's going on here? If there's a follow-up task let's file an issue and add a link to that issue in this comment.

Reply:
Now that ATen has a sort method with the stable argument, I could call the stable version here. Did not do it earlier b cause I did not know whether schemas were right.

Reply:
I see. I think your schemas are correct and we should either update this comment or call it.